docs/api/spaces-management/get_shareable_references.asciidoc

```diff
@@ -0,0 +1,81 @@
+[role="xpack"]
+[[spaces-api-get-shareable-references]]
+=== Get shareable references API
+++++
+<titleabbrev>Get shareable references</titleabbrev>
+++++
+
+experimental[] Get shareable references.
+
+Collects references and spaces context for saved objects.
+
+[[spaces-api-get-shareable-references-request]]
+==== {api-request-title}
+
+`POST <kibana host>:<port>/api/spaces/_get_shareable_references`
+
+[[spaces-api-get-shareable-references-request-body]]
+==== {api-request-body-title}
+
+`objects`::
+  (Required, object array) The saved objects to collect outbound references for.
++
+.Properties of `objects`
+[%collapsible%open]
+=====
+  `type`:::
+    (Required, string) The saved object type.
+
+  `id`:::
+    (Required, string) The saved object ID.
+=====
+
+[role="child_attributes"]
+[[spaces-api-get-shareable-references-response-body]]
+==== {api-response-body-title}
+
+`objects`::
+  (object array) The returned input object or one of its references, with additional context.
++
+.Properties of `objects`
+[%collapsible%open]
+=====
+  `type`:::
+    (string) The saved object type.
+
+  `id`:::
+    (string) The saved object ID.
+
+  `originId`:::
+    (string) The origin ID of the referenced object (if it has one).
+
+  `inboundReferences`:::
+    (object array) References to this object.
++
+NOTE: This does not contain _all inbound references everywhere_, it only contains inbound references to this object within the scope of this operation.
++
+.Properties of `inboundReferences`
+[%collapsible%open]
+======
+    `type`::::
+      (string) The type of the object that has the inbound reference.
+
+    `id`::::
+      (string) The ID of the object that has the inbound reference.
+
+    `name`::::
+      (string) The name of the inbound reference.
```

Comment:
For my own understanding, is this the name of the reference (are references named?) or the name of the object that has the inbound reference?

Reply:
I've not been able to get an example response that includes inboundReferences but my assumption would be that this is the name of the reference itself like in this index pattern from the example data set:

```json
    {
      "id": "ff959d40-b880-11e8-a6d9-e546fe2bba5f",
      "name": "indexpattern-datasource-current-indexpattern",
      "type": "index-pattern"
    },
```